README.md

```diff
@@ -25,8 +25,14 @@ On [snap enabled](https://snapcraft.io/docs/core/install) systems, the latest pr
 ```bash
 sudo snap install openspades
 ```
+#### Flatpak package
+On [flatpak enabled](https://flatpak.org/setup/) systems, OpenSpades can be installed with:
 
-Once installed, you'll be able to launch OpenSpades from inside the desktop menu or from your terminal with the `openspades` command.
+```bash
+flatpak install flathub jp.yvt.OpenSpades
+```
+
+Once installed, you'll be able to launch OpenSpades from inside the desktop menu or from your terminal with the `openspades` (Snap) or `flatpak run jp.yvt.OpenSpades` (Flatpak) command.
```

Comment:
It makes more sense to mention about Snap in its own (already available) section. 

Adding:
> Once installed, you'll be able to launch OpenSpades from inside the desktop menu or from your terminal with the `openspades`

To below the installation instructions just like the Flatpak should do it.

Reply:
Thanks for the feedback, I've updated the README now.